projects/kavaswap.js

```diff
@@ -2,7 +2,19 @@ const utils = require('./helper/utils');
 
 async function tvl(timestamp, ethBlock, chainBlocks) {
     let balances = {};
-    const response = await utils.fetchURL('https://api.data.kava.io/swap/pools');
+    let startBlock = 4698;
+    const dailyBlockInterval = 12495;
+    let startDate = new Date(Date.UTC(2021, 7, 31));
+    let dateNow = new Date(timestamp * 1000);
+
+    while (startDate < dateNow) {
```

Comment:
maybe we could try to estimate this only when getting historical data, in other cases we could just get the most recent data